tests/test_cli.py

```diff
@@ -202,6 +202,29 @@ def test_deploy_one_new_environment_variable(get_client, runner):
     assert u'Deployment successful' in result.output
 
 
+@patch('ecs_deploy.cli.get_client')
+def test_deploy_one_new_secrets_variable(get_client, runner):
+    get_client.return_value = EcsTestClient('acces_key', 'secret_key')
+    result = runner.invoke(cli.deploy, (CLUSTER_NAME, SERVICE_NAME,
+                                        '-s', 'application', 'baz', 'qux',
+                                        '-s', 'webserver', 'baz', 'quux'))
+
+    assert result.exit_code == 0
+    assert not result.exception
+
+    print(result.output)
```

Comment:
I guess this was for debugging purposes?

Reply:
Ah whoops, will fix that up